lightwood/mixers/helpers/default_net.py

```diff
@@ -92,7 +53,7 @@ def __init__(self, ds, dynamic_parameters, shape=None, selfaware=False, size_par
 
             layers = []
             for ind in range(len(shape) - 1):
-                linear_function = PLinear  if CONFIG.USE_PROBABILISTIC_LINEAR else torch.nn.Linear
+                linear_function = PLinear if CONFIG.USE_PROBABILISTIC_LINEAR else torch.nn.Linear
```

Comment:
let's move the probabilistic linear to the params

Reply:
I think that, as it stands, Plinear is still a "dev" feature only, so I'd rather leave it in the CONFIG (See my comment about wanting to leave dev flags in the CONFIG because it's easy for us to switch them around when we play with stuff)

Reply:
But if you'd prefer I can move it, I have no strong opinion either way.